### PR TITLE
Feature/remove account updated

### PR DESCRIPTION
**Pull request by Ecenursahin**
This is the updated branch for remove account, the previous branch had a lot of refactoring conflicts hence this is the mostt upto date one to add to the main.

### CODE REVIEW

File: src/main/java/server/bank/NewBank.java [hunk 1]
```diff
@@ -230,7 +232,52 @@ public String createAccount(Customer customer, BufferedReader in, PrintWriter ou
...
+			return "Exit request is taken, going back to the main menu.";
+			}
+			
+		 else {
+				for (int i = 0; i < customerAccounts.size(); i++) {
+					if (customerAccounts.get(i).getAccountNumber().equals(accountNumber)) {
+						Account customerAccount = customerAccounts.get(i);
+						Double currentBalance = customerAccount.getBalance();
+						if(currentBalance!=0) {
+							return String.format("Process failed. The outstanding balance is not 0.");
```
Comment (bnaylor1989) [inline src/main/java/server/bank/NewBank.java:266]: You don't need String.format here, you v=can just return the string i.e. 
return "Process failed. The outstanding balance is not 0.";
Also Could you just update this mesage slightly to say something like, Account removal failed. The outstanding balance is not 0.

File: src/main/java/server/user/Customer.java [hunk 2]
```diff
@@ -55,5 +56,11 @@ public void addAccount(Account account) {
 		accounts.add(account);
 		// TODO: save new account details to the database
 	}
+	
+	public void removeAccount(Account account) {
+		accounts.remove(account);
+		int accountNum = Integer. parseInt(account.getAccountNumber());
```
Comment (bnaylor1989) [inline src/main/java/server/user/Customer.java:62]: Can you get rid of the space after Integer.

**Review by bnaylor1989 — APPROVED** [review]
Hey Ece, this is good stuff. Just a couple of minor things to address. But happy to approve.

File: src/main/java/server/bank/NewBank.java (hunk 1 above)
Comment (Ecenursahin) [reply src/main/java/server/bank/NewBank.java:266]: Thanks Ben, doing the update now!

File: src/main/java/server/user/Customer.java (hunk 2 above)
Comment (Ecenursahin) [reply src/main/java/server/user/Customer.java]: Sure!

**Review by bnaylor1989 — APPROVED** [review]
Looks great. All good to merge.

**Comment by Ecenursahin** [comment]
Thank you so much Ben!

**Pull request merged**